### ISSUE TITLE
[Feature Request] Select which element to scroll

### ISSUE THREAD

**Issue by danman113** [danman113/simple-autoscrollv2#2]
Many users have the problem where they want to scroll an iframe, or some scrolling element in a website.

I could support these usecases by adding some kind of "Picker" element, so users can select which element they want to scroll on a page.

UX for this could be similar to the inspect element tool in developer tools
![image](https://user-images.githubusercontent.com/8608120/232978720-8b37aa84-e352-4089-b591-6a234f8e66cd.png)

Or more likely, the "Picker" tool from ublock origin
![image](https://user-images.githubusercontent.com/8608120/232979029-2672213a-ba33-4ba8-aafa-2228daf55cac.png)

**Comment by R2bEEaton** [comment]
This would be great!

**Comment by CuddlyPaws** [comment]
hi @danman113 the shortcut key is not working
![image](https://github.com/danman113/simple-autoscrollv2/assets/118006987/0cc3f822-a8ce-4d2a-add0-a34597920338)
 i set this shortcut key but whenever i press this ctrl+U it does not scroll anything
 but once i click on the extension and click go,after that the shortcut key works fine.
 it's just that the first time it don't do anything.
 you have to click the extension and then the key will work further in website
 and one more thing one you refresh the page. the key doesn;t work.
 to summarise, if we click extension -> Go , after that only the keys works very fine
 
I please request you to see this isse/bug and fix it or tell us something we can do.
It's so painful in every new website we have to extension-> Go to work for the key

thanks ,
have a nice day

**Comment by danman113** [comment]
@Himan16r just to confirm are you using Google chrome or Microsoft Edge? I haven't explicitly added any Edge support at this time

**Comment by CuddlyPaws** [comment]
> @Himan16r just to confirm are you using Google chrome or Microsoft Edge? I haven't explicitly added any Edge support at this time

umm yup that's why
i m using edge from the last 5  years no chrome for me
hopefully u will add some support for edge too in future
thnx @danman113

**Comment by plxel** [comment]
looks like still not supported

made simple patch with input field to write selector for scrollable container:

fork here https://github.com/plxel/simple-autoscrollv2

<img width="173" height="202" alt="Image" src="https://github.com/user-attachments/assets/bc96f286-3465-4ef9-8306-79836569e208" />
